src/Raven.Client/Documents/Session/Operations/LoadOperation.cs

```diff
@@ -107,6 +128,9 @@ public void SetResult(GetDocumentsResult result)
                 return;
 
             _session.RegisterIncludes(result.Includes);
+            
+            _session.RegisterCounters(result.Counters, _includeAllCounters, _ids);
+            _session.RegisterMissingCounters(_ids, _counters);
```

Comment:
Pretty sure that this can be merged into a single operation. You just need to pass the `_counters` to the `RegisterCounters`.

Reply:
Okay will merge